### ISSUE TITLE
[주간회의] 10월 12일

### ISSUE THREAD

**Issue by unizard** [kangdain/TLO#11]
!!

**Comment by unizard** [comment]
@kangdain 
앞으로 한일 & 할일 함께 적어주세요~
오늘 이야기 했던 부분은, Tensorflow + TensorRT + Go language + Dash Board(시각화툴) 세트예요.
시간될때 한번 알아보세요.

**Comment by kangdain** [comment]
###한일
- 1. Lab08,Lec9-1  : Tensormanipulation :  Shape ,Rank , Axis 예제 및 shape, rank 다르더라도 tensor 연산이 가능한 broadcasting과 그외 많이 사용되는 reshape, one hot, reduce_mean 등의 연산을 알 수 있었다.  2.제 4차 청년 TLO 교육   (월) 
-  공휴일 (화)
- Vue.js Form input 핸들링을 위한 모듈을 수행해 봄으로써 프로제트 구성에 필요할 요소들을 익혀볼 수 있는 시간을 가졌다.(수) 
- Lex9-x ,Lec9-2: 미분의 기본 개념 및 실제값과 예측값 사이에서 나타난 오류(= error, cost ) 를 통해 미분값을 계산하는 Backpropagation이 실제로 어떻게 동작하는 지에 대해 이해하기위하여 chain rule을 배웠다. (목) 	
- 1.Lab9-1,Lab9-2 : xor을 위한 딥러닝 / TensorBoard 사용법 2. Filters & Mixins :Vue.js  Filter를 통해 데이터를 체이닝 하는 방법과 Mixin 으로 반복되는 기능들을 분리 시켜  전역에서 재사용 가능 하도록 만들수 있다.(금)  

###할일

월,화: 1. html&css 참고 문헌 한번씩 훑어 보기,   2. vue .js 기본 개념 마무리
수 ~ 금: home.vue 완성
